components/navbar/navbar.snip.html

```diff
@@ -77,9 +78,17 @@
           {{> ../calltracking/calltracking.snip.html}}
         {{/calltracking}}
       </div>
+      {{#state}}
```

Comment:
i prefer the ternary operator :) even thought it is a bit nasty - ill open up a FR to the amphtml to make it simpler to toggle classes via bind